test/per_file_coverage.sh

```diff
@@ -29,7 +29,9 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/common/tap:94.6"
 "source/extensions/common/wasm:95.3" # flaky: be careful adjusting
 "source/extensions/common/wasm/ext:92.0"
+"source/extensions/filters/common:95.3"
 "source/extensions/filters/common/fault:94.5"
+"source/extensions/filters/common/lua:87.2"
```

Comment:
I couldn't figure out how to add coverage on the remaining conversions. Would appreciate any insights. I can also just remove these cases and only support a few we actually need.